.github/workflows/test-spyre.yml

```diff
@@ -129,6 +132,9 @@ jobs:
           MASTER_ADDR: localhost
           DISTRIBUTED_STRATEGY_IGNORE_MODULES: WordEmbedding
         run: |
+          # Delete the source code so we can ensure we're testing the installed
+          # wheel
+          rm -fr vllm_spyre
           source .venv/bin/activate
-          uv run pytest ${{ matrix.test_suite.flags }} \
+          python3 -m pytest ${{ matrix.test_suite.flags }} \
```

Comment:
`uv run` will re-sync by default which uninstalls the `vllm_spyre` wheel :(

We could do a `--no-sync` here, though I think the explicit `source .venv/bin/activate / python -m ...` is more familiar

Reply:
makes sense, though I needed to read your comments 3 times and scroll through earlier build logs to grasp what's happening 🫤

Reply:
me too!